Source/WebKit/UIProcess/Cocoa/Foundation+Extras.swift

```diff
@@ -21,10 +21,13 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 // THE POSSIBILITY OF SUCH DAMAGE.
 
-#if ENABLE_SWIFTUI && compiler(>=6.0)
+#if compiler(>=6.0)
 
 import Foundation
 
+typealias String = Swift.String
+typealias URL = Foundation.URL
```

Comment:
So to access `WTFString.h`, we would want to use `WTF::String`?

Reply:
to access it from Swift (which you probably wouldn't need to do at all) you would do `wtf.String` (wtf being the module name)

Reply:
okay! Just curious how the syntax looks like